### DISCUSSION TITLE
双4K屏幕轨迹绘制延迟很严重

### DISCUSSION THREAD

**Discussion by hhdebb** [shuax/MouseInc.Settings#245] [Q&A]
- 单屏4K没啥问题，最近接了个副屏也是4K的，使用就延迟很严重，无奈只能先关闭轨迹暂时不显示了。

**Comment by shuax** [comment]
没有办法，程序绘制4K全屏手势太慢了。

**Reply by tangjyan** [reply]
不知道WGestures是怎么解决的，没有这个问题。

**Reply by ixukai** [reply]
原来如此，今天换了4K屏幕，也发现了这个问题，总感觉鼠标有点飘，不跟手。。。

**Reply by CaiSijian** [reply]
能考虑一下与特定手势相对应的特定热键吗？
就是只有特定应该程序才有的热键
